docs/interactions/Slash_Commands.md

```diff
@@ -871,3 +876,4 @@ Not all message fields are currently supported.
 | content           | string                                                   | message content                                                                             |
 | embeds?           | array of [embeds](#DOCS_RESOURCES_CHANNEL/embed-object)  | supports up to 10 embeds                                                                    |
 | allowed_mentions? | allowed mentions                                         | [allowed mentions](#DOCS_RESOURCES_CHANNEL/allowed-mentions-object) object                  |
+| flags | int | set to `64` to make your response ephemeral |
```

Comment:
```suggestion
| content?          | string                                                   | message content                                                                             |
| embeds?           | array of [embeds](#DOCS_RESOURCES_CHANNEL/embed-object)  | supports up to 10 embeds                                                                    |
| allowed_mentions? | allowed mentions                                         | [allowed mentions](#DOCS_RESOURCES_CHANNEL/allowed-mentions-object) object                  |
| flags?            | int                                                      | set to `64` to make your response ephemeral                                                 |
```
- `content` is optional now bc it isn't required for deferred responses
- `flags` is always optional

Reply:
(FWIW it was never actually required -- like in the Figma, you can send only embeds and no content)

Reply:
No, in my case I have to set content as '" or it just gives me an error.